src/main/resources/templates/add.ftl

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<body>
+<h3>Adding new blog</h3>
+<a href="http://localhost:8080/">Home</a>
+<a href="http://localhost:8080/blog/search/">Search blog</a>
```

Comment:
You need to have all navigation items like "Home", "Search blog", "Add blog" and etc. at any point of time, because it's a main navigation on your website and it should stay persistent